apps/system/js/app_window_manager.js

```diff
@@ -44,6 +44,10 @@
       }
       if (active) {
         this.focus();
+      } else if (!UtilityTray.shown) {
```

Comment:
Never a good practice to know who is on top of appWindow. Please file follow up to fix it. Also this is wrong if attention window is opened and I don't want see <code>if (!attentionWindowManager.isActive && UtilityTray.isActive)</code> here.

The ideal way is have hierarchyManager to broadcast focus and ignore the handler in utilityTray.